src/SourceBuild/content/eng/pipelines/templates/jobs/sdk-diff-tests.yml

```diff
@@ -165,3 +169,13 @@ jobs:
       mergeTestResults: true
       publishRunAttachments: true
       testRunTitle: $(Agent.JobName)
+
+  - ${{ if eq(parameters.publishTestResultsPr, 'true') }}:
+    - ${{ if or(eq(variables['Build.SourceBranchName'], 'main'), contains(variables['Build.SourceBranchName'], 'release')) }}:
+      - template: ../steps/publish-test-results-pr.yml
+        parameters:
+          pipelineMode: sdk
+          repo: dotnet/installer

```

Comment:
Am I correct in understanding this is going to open public GH PRs? Are there cases in which this should not happen because it might disclose something?  (e.g. this is running against an internal release branch)?

Reply:
> Am I correct in understanding this is going to open public GH PRs?

Correct.

> Are there cases in which this should not happen because it might disclose something? 

I have this conditionalized the run against any main, internal/release/* or release/* branch. If we think that having this run on the internal branches might be an issue, I can change it to run only on the main and release/* branches.

Reply:
If this is triggered for `internal/release/*`, it should create PRs in AzDO, not GH. So I would say internal should be removed for now. If later we decide it will be useful that this support internal branches, then we can add support for that.